src/code/utilities/url-params.js

```diff
@@ -12,4 +12,16 @@ var urlParams;
        urlParams[decode(match[1])] = decode(match[2]);
 })();
 
+export function getClassId() {
+  // Pull the class number out of the class info
+  let regex = /classes\/(\d*)/g,
+      matches = regex.exec(urlParams.class_info_url);
+
+  return matches ? matches[1] : null;
+}
+
+export function getUserId() {
+  return urlParams.domain_uid;
```

Comment:
Just for my own understanding: do you know why the user_id is in the url as `domain_uid`?

Also, the classId above is pulling a number from a property called `classes` -- are we sure this won't be a list of numbers if the user is in more than one class? Will this still work correctly?